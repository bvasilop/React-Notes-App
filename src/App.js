import React from 'react';
import DarkModeToggle from './DarkModeToggle';
import Content from './Content';
import NoteApp from './NoteApp';

import './styles.scss';

function App() {
  return (
    <div>
      <div className="navbar">
        <DarkModeToggle />
      </div>
      <Content />
      <NoteApp />
    </div>
  );
}

export default App;
