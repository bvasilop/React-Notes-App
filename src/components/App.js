import React from 'react';
import NoteApp from './NoteApp';

const App = () => (
  <div>
    <NoteApp />
  </div>
);

export default App;
