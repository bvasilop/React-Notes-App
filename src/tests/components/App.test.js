import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import '../setupTests';
import App from '../../components/App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
