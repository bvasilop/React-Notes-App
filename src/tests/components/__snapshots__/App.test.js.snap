// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <div
    className="navbar"
  >
    <DarkModeToggle />
  </div>
  <Content />
  <NoteApp
    appName="React Note App"
    placeholder="Add some notes!"
    subHeader="Notes"
    subHeader2="Add Note"
  />
</div>
`;
